CHANGES.md

```diff
@@ -98,7 +98,7 @@ construction (instance creation... along with any constructor).
 ### Added JSR223 javax.script adapter engine implementation in a new bsh.engine package
 
 This implementation of the spec supports pluggable ScriptContext.  Please
-see the TestBshScriptEngine.java file (and the spec) for a starting point
+see the `TestBshScriptEngine.java` file (and the spec) for a starting point
```

Comment:
Might want to link to the actual file. Using a relative link like so: `[TestBshScriptEngine.java](src/test/java/bsh/TestBshScriptEngine.java)`

Reply:
Good idea, I have linkified all of the references to .java files.